baselines/FedSMOO/FedSMOO/utils.py

```diff
@@ -0,0 +1,22 @@
+"""Define any utility function.
+
+They are not directly relevant to  the other (more FL specific) python modules. For
+example, you may define here things like: loading a model from a checkpoint, saving
+results, plotting.
+"""
+
+import matplotlib.pyplot as plt
+from typing import List, Tuple
+
+def plot_fn(cfg, acc: List[Tuple[int, float]]):
+    x_val = [x[0] for x in acc]
+    y_val = [x[1] for x in acc]
+
+    plt.plot(x_val,y_val,'or')
```

Comment:
```suggestion
    plt.plot(x_val,y_val,'or')
    plt.plot(x_val,y_val,'-r')
    plt.grid()
```

Reply:
Probably you are still working on this. I addition to those changes, don't forget to add meaningful labels for both axis and a title.